lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,38 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
+          # binding.pry
+          options[:destination_branch] ||= 'master'
+
+          message = "#{options[:message]}"
+
+          if "#{options[:pull_request_number]}".length > 0
+            message << "\nMerges ##{options[:pull_request_number]}\n"
+          end
+
+          if lgtm_authors = Array(options[:lgtm_authors]) and lgtm_authors.any?
+            message << "\nLGTM given by: @#{lgtm_authors.join(', @')}\n"
+          end
+
+          data = {
+            "commit_title" => options[:title],
+            "commit_message" => options[:message],
+            "sha" => options[:head].sha,
+            "squash" => true
+          }
+          
+          res = GitReflow::GitServer::GitHub.connection.pull_requests.merge(
+            "#{GitReflow::Config.get('github.owner')}", 
+            "#{GitReflow::Config.get('github.repo')}", 
+            "#{options[:pull_request_number]}", 
+            data
+          )
+
+          append_to_squashed_commit_message(message) if message.length > 0
+          return res
```

Comment:
Since the response is specific to GitHub, I would rather see the handling of the failure status codes here.  If it is anything but `response.ok` we should raise an error and catch the exception elsewhere (by default, if the error is not caught, this will cause reflow to immediately exit, which I think is ideal).